### DISCUSSION TITLE
Vertical Slice Architecture Implementation

### DISCUSSION THREAD

**Discussion by garthtrickett** [Rykuno/fullstack-example#2] [Show and tell]
I've been following along with @Rykuno 's updates and adding them into a VSA version of this repo

https://github.com/garthtrickett/ci-eng

**Comment by Rykuno** [comment]
Oh this is really cool, ty. This stack kinda popped off way more than I thought and I have quite a few DM's about it - but I have my own side businesses and day job to keep track of so its been hard to respond to most.

I decided to make a website, add versioning, a CLI tool, and actual documentation. I have ~30 pages completed and I'll release it in a couple weeks; time permitting.
